pkg/test/toolchain/toolchain.go

```diff
@@ -22,3 +24,11 @@ func NewCheInstallation(cheNamespace string) *v1alpha1.CheInstallation {
 func GenerateName(prefix string) string {
 	return fmt.Sprintf("%s-%d", prefix, time.Now().UnixNano())
 }
+
+func NewTektonInstallation() *v1alpha1.TektonInstallation {
+	return &v1alpha1.TektonInstallation{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: TektonInstallation,
```

Comment:
Not sure but is it fine/mandatory to have same name for `TektonInstallation` ??  At least for test, we can create multiple `TektonInstallation` with different name so that if test running parallel then they are not using each other `TektonInstallation` CR .. right ??

Reply:
No, we need to keep this name same, to have only one resource across cluster scoped

Reply:
> No, we need to keep this name same, to have only one resource across cluster scoped

But that should be applied for the `Cheinstallation` as well, which is not the case as you generate the name there.

Reply:
sure, it'll come when we are refactoring cheInstallation controller

Reply:
But you already changed that name and the part of the code, so why doing just half of the changes here...?
Anyway, I will trust you that you will change it in the next PR